gallery/src/pageConfig.ts

```diff
@@ -179,7 +181,9 @@ const pageConfig: PageConfig = {
     'Tag': { content: NxTagPage, type: 'react' },
     'Small Tag': { content: NxSmallTagPage, type: 'react' },
     'Threat Counter': { content: NxThreatCounterPage, type: 'react' },
+    'Vulnerability Counter': { content: NxVulnerabilityCounterPage, type: 'react' },
     'Small Threat Counter': { content: NxSmallThreatCounterPage, type: 'react' },
+    'Small Vulnerability Counter': { content: NxSmallVulnerabilityCounterPage, type: 'react' },
```

Comment:
I'd rather not commingle these with the Threat-related components.  Move them to the bottom of the list where "V" would order naturally.

Reply:
Modified the list here: [d2833e7](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/d2833e7c2c1cc62ab5e8b0130c8e8809788dd520)

Reply:
> Modified the list here: [d2833e7](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/d2833e7c2c1cc62ab5e8b0130c8e8809788dd520)

They should be after Overflow Tooltip

Reply:
> > Modified the list here: [d2833e7](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/d2833e7c2c1cc62ab5e8b0130c8e8809788dd520)
> 
> They should be after Overflow Tooltip

Fixed that here: [ad27aff](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/ad27affbabad93f49ac13a8761dffbda7d57a9a5)